modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1180 @@
+// TINY-9231
+
+Dragging or dropping block `links` into other `links` was producing invalid nesting of other transparents such as `anchors`. As a result, these transparent elements would produce `anchor` tags within other `anchor` tags, which is not a general practice. As an example `links` and `anchors` that are defined to an element, must not be nested within or contain any other element and vice versa.
+
+Changes were implemented in {productname} 6.4 to fix this issue to ensure that any invalid child anchor elements that try to reproduce this action are unwrapped when the editor verifies another transparent has been dropped into another transparent (anchor) element.
+
+In {productname} 6.4, dragging content such as `links` into other `links` will now produce *valid* HTML `output` after a drag operation occurs which will be consistent across all browsers.
+
+=== The editor.insertContent API would insert contents inside noneditable elements if the selection was inside the element
+// TINY-9462
+
+In previous versions of {productname}, the settings for the `contentEditable` property in the editor root was set to `false`. As a consequence, the user could insert content using `api` commands to edit a `contenteditable="false"` root element.
+
+To fix this bug, in {productname} 6.4, a blocker was added to the `editor.insertContent` api and `mceInsertContent` command, which stops `contenteditable="false"` root elements from being edited.
+
+In {productname} 6.4, inserting contents inside a `contenteditable="false"` element while the selection is inside another element, is no longer possible.
+
+=== Closing a dialog would scroll down the document in Safari
+// TINY-9148
+
+While using previous versions of {productname}, `clicking` away to dismiss the *Search* and *Replace* dialog within Safari would scroll the page to the bottom of the editor. As a result the user would have to manually scroll back to the original `focus` area each time the steps above were executed.
+
+To correct this issue, a new Css.set was implemented to keep the `focus` on the DOM container in the initial scroll document position. 
+
+In {productname} 6.4, when the user `clicks` away to dismiss the *Search* and *Replace* dialog within Safari, the focus now returns to the editor and the scroll does not change position within the editor.
+
+=== Inline headers would not work in some situations when the editor was moved too far right horizontally
+// TINY-8977
+
+// Cause
+// Consequence
+// Fix
+// Result
+
+=== Quick toolbars were incorrectly rendered during the dragging of contenteditable="false" elements
+// TINY-9305
+
+The quickbar was displaying incorrectly during a drag and drop `action` for contenteditable="false" elements. As a consequence, during the movement of a contenteditable="false" element, the selection was updated, which triggered a *NodeChange*. This selection update would cause an absence of the caret location during a dragging or dropping event.
```

Comment:
```suggestion
The quickbar was displaying incorrectly during a drag and drop `action` for contenteditable="false" elements. As a consequence, during the movement of a contenteditable="false" element, the selection was updated, which triggered a *NodeChange* event. This selection update would cause an absence of the caret location during a dragging or dropping event.
```